recipes/CommonLanguage/lang_id/train.py

```diff
@@ -296,13 +296,15 @@ def label_pipeline(language):
         checkpointer=hparams["checkpointer"],
     )
 
+    torch.backends.cudnn.benchmark = True
+
     # The `fit()` method iterates the training loop, calling the methods
     # necessary to update the parameters of the model. Since all objects
     # with changing state are managed by the Checkpointer, training can be
     # stopped at any point, and will be resumed on next call.
     lid_brain.fit(
         epoch_counter=lid_brain.hparams.epoch_counter,
-        train_set=datasets["train"],
+        train_set=datasets["dev"],
```

Comment:
This looks like a temporary chance to train everything faster . It should be datasets["train"], right?